README.md

```diff
@@ -1,2 +1,72 @@
 # pds-doi-ui
 web UI for pds-doi-service
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
Can you make the README a bit more specific.

Especially, we are missing the `npm install` as a first step I think.

Reply:
Sure thing! This readme was auto generated by react but I can go through and make it more specific.